### DISCUSSION TITLE
Slack Group for Translations

### DISCUSSION THREAD

**Discussion by 0xtelmo** [reactjs/reactjs.org-translation#243] [Team Posts]
Hey all,

I thought it would be useful to have a Slack Group to discuss some topics.

The idea is to keep discussions related to PRs in GitHub but can be useful to discuss other stuff, such as what words we should NOT translate, etc.

What do you all think?

**Comment by jakubdrozdek** [comment]
Hey, I've created a Spectrum community for the Polish i18n team, but I can create channels for other teams. Feel free to join if you want:
https://spectrum.chat/reactjs-org-i18n

**Comment by arshadkazmi42** [comment]
This is a really great idea

**Comment by 0xtelmo** [comment]
Whoever wants to join the Slack Group drop me your email 👍

**Comment by WendellAdriel** [comment]
Hey @telmogoncalves, you could create an "auto-invite" page using something like this:
https://github.com/outsideris/slack-invite-automation

Deploy on Heroku for free and send the link here!!! 😉

**Comment by 0xtelmo** [comment]
> Hey @telmogoncalves, you could create an "auto-invite" page using something like this:
> https://github.com/outsideris/slack-invite-automation
> 
> Deploy on Heroku for free and send the link here!!! 😉

I'll do that once I have a little free time, thanks 👍

**Comment by jakubdrozdek** [comment]
Since I'm already used to Slack + it has more readable discussion threads, I'd vote for Slack. Let me ask the Polish team if they also would like to join.

@telmogoncalves Okay, where do I drop my email? I don't want it to be public or anything.

**Comment by 0xtelmo** [comment]
> Since I'm already used to Slack + it has more readable discussion threads, I'd vote for Slack. Let me ask the Polish team if they also would like to join.
> 
> @telmogoncalves Okay, where do I drop my email? I don't want it to be public or anything.

You can drop me a message on [Twitter](https://twitter.com/telmo) if that works for you.

**Comment by 0xtelmo** [comment]
Here's the link for the invitations: https://rt-slack.herokuapp.com/

Guess it's working, otherwise, let me know 🤞

**Comment by 0xtelmo** [comment]
@WendellAdriel want to give it a try?

**Comment by WendellAdriel** [comment]
![image](https://user-images.githubusercontent.com/11641518/52719690-0531d180-2f9e-11e9-94a1-98d02abb967f.png)
I got this @telmogoncalves

**Comment by 0xtelmo** [comment]
Will try to fix that ASAP.

**Comment by 0xtelmo** [comment]
Here it is: https://rt-slack-invite.herokuapp.com

**Comment by WendellAdriel** [comment]
All good now @telmogoncalves!!! 😉

**Comment by tesseralis** [comment]
This slack channel has received the tesseralis seal of approval 👍
